src/loaders/yaml.cpp

```diff
@@ -163,10 +163,10 @@ bool loadFromYAMLNode(const YAML::Node& node, ReaderWriter& config, const Resolv
 #ifdef YAML_VERSION_0_3
             for(YAML::Iterator it2 = n.begin(); it2 != n.end(); ++it2)
 #else
```

Comment:
We should either get rid of this old code or change it too.

Reply:
done

Reply:
I think the new code does work both the old(0.3) and current versions. Therefore it is safe to remove this `if/else`.